conf/gunicorn-dev.conf.py

```diff
@@ -7,8 +7,12 @@
 timeout = 0
 
 
-if 'H_GUNICORN_CERTFILE' in environ:
-    certfile = environ['H_GUNICORN_CERTFILE']
+max_requests = 100
```

Comment:
`GUNICORN_CMD_ARGS` is an alternative to this but I find changing env vars in production bfinifnyc and error prone. This is also reviewable.

We could use the env var if we needed different values in different servers. Do we need that already in Canada? I don't think we've seen memory issues there.